README.md

```diff
@@ -52,16 +52,16 @@ Typical configuration and installation
     meson compile -C builddir/
     sudo meson install -C builddir/
 
+For a release-oriented build, recommended options are kept in the `build-release.ini`
```

Comment:
I think it's aimed to what we developers think a distro could use. Or at least closer to it for us to use when e.g. comparing size/speed etc. And it can be used by end users too. Not sure about a better word.

Reply:
Basically what we're saying it "these are the defaults", which brings the question - why don't we adjust the defaults in the meson build itself?

If we don't want to go there, let's just spell out the main target audiences `s/For a release-oriented build/For end-user and distributions builds/` and keep the name as-is.

Reply:
changing the defaults might be a good option

Reply:
Opened https://github.com/kmod-project/kmod/pull/225

Feel free to mix and match commits from here + ^^ as suitable.